app/models/conference_user.rb

```diff
@@ -15,6 +15,6 @@ def role_is_valid
   end
 
   def user_role_is_crew
-    self.errors.add(:role, 'User role is not crew') unless self.user.is_crew?
+    self.errors.add(:role, 'User role is not crew') unless self.user and self.user.is_crew?
```

Comment:
I'd be interested in a stack trace for this. A conferences user should never be `nil`. Sadly AR validations are not executed in order.